src/protocols/at-ps2/mouse_interface.c

```
@@ -403,38 +438,42 @@ void mouse_input_event_handler() {
-          mouse_id = ATPS2_MOUSE_ID_UNKNOWN;               // Reset Mouse ID
-          mouse_state = INIT_AWAIT_ACK;  // Set State to Await Acknowledgement
-          detect_stall_count = 0;        // Reset Stall Counter
-          mouse_command_handler(ATPS2_CMD_RESET);   // Send Reset Command
-        }
-      } else if (mouse_state == UNINITIALISED) {
-        LOG_DEBUG("Awaiting mouse detection. Please ensure a mouse is connected.\n");
-        detect_stall_count =
-            0;  // Reset the detect_stall_count as we are waiting for the clock to go HIGH.
-      }
+    // Guard against uninitialised PIO engine
+    if (pio_engine.pio == NULL)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)